packages/atclient/include/atclient/connection.h

```diff
@@ -21,6 +21,7 @@ typedef struct atclient_connection {
   mbedtls_ctr_drbg_context ctr_drbg;
 } atclient_connection;
 
+// TODO: where can ctx be made a const * ?
```

Comment:
I'm still establishing this, but I don't think it can be, simply due to the underlying mbedtls ssl context needing read / write permissions.